### ISSUE TITLE
About loss weight

### ISSUE THREAD

**Issue by Leon-Li-1** [KaiqiangXiong/CL-MVSNet#11]
Hello author, Thanks for your excellent work!
I'm trying to reproduce your code. I trained on an NVIDIA 4090 and the overall I got was 0.398. Then I found that the weight of the loss function in your code is different from that in the paper. I would like to ask how I should set the weights of the loss functions of each part.
in code -> self.unsup_loss = self.args.wrecon * self.reconstr_loss + 6 * self.ssim_loss + 0.18 * self.smooth_loss
but, in paper -> wrecon= 0.8,  wssim = 0.2, wsmooth=0.0067.

**Comment by KaiqiangXiong** [comment]
Regarding the loss weights, the settings in the code represent the final version. Additionally, for general MVS networks, it has been observed that training the same code on different GPUs often yields different results, especially when transitioning from V100/A100 to 3090/4090. Perhaps tuning the hyperparameters could address this issue.

**Comment by Leon-Li-1** [comment]
Got it. Thank you for your guidance. I want to know at what values the final training loss and test loss are for normal training?













At 2025-05-14 11:29:36, "Kaiqiang Xiong" ***@***.***> wrote:

KaiqiangXiong left a comment (KaiqiangXiong/CL-MVSNet#11)

Regarding the loss weights, the settings in the code represent the final version. Additionally, for general MVS networks, it has been observed that training the same code on different GPUs often yields different results, especially when transitioning from V100/A100 to 3090/4090. Perhaps tuning the hyperparameters could address this issue.

**Comment by KaiqiangXiong** [comment]
These training logs were not retained. I recall that when it finally converged, the percentage of the pixels with over 2mm depth error was around 0.256-0.260. We will determine whether it has converged based on this metric on the validation set.

**Comment by Leon-Li-1** [comment]
Got it. Thank you very much!
















At 2025-05-14 14:41:54, "Kaiqiang Xiong" ***@***.***> wrote:

KaiqiangXiong left a comment (KaiqiangXiong/CL-MVSNet#11)

These training logs were not retained. I recall that when it finally converged, the percentage of the pixels with over 2mm depth error was around 0.256-0.260. We will determine whether it has converged based on this metric on the validation set.
